README.md

```diff
@@ -92,31 +94,31 @@ See [prelease.md](docs/prerelease.md)
 
 [We decided](https://github.com/trussworks/react-uswds/issues/2001) to focus on our [storybook](https://trussworks.github.io/react-uswds/) over hosting an example/boilerplate implementation, but you can find a few such things from our friends:
 
-* [NASA's Americana Remix stack](https://github.com/nasa-gcn/americana-stack?tab=readme-ov-file)
-* [USDS's Gatsby starter template](https://github.com/usds/gatsby-uswds-ts-starter)
-* [Nava's Next.js starter template](https://github.com/navapbc/template-application-nextjs)
+- [NASA's Americana Remix stack](https://github.com/nasa-gcn/americana-stack?tab=readme-ov-file)
```

Comment:
Looks like prettier is opinionated on this and since I edited this file, the pre-commit check picked it up